code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -438,24 +446,28 @@
 	desc = "A classic brown letterman jacket. Looks pretty hot and heavy."
```

Comment:
why can't golems wear overcoats and/or puffer jackets?

at the very least, bone golems should be allowed to wear puffer jackets...

Reply:
Not really any excuse tbh. I'll fix that up I just wanted to get the ball rolling with a few choice examples.